code/_onclick/hud/rendering/render_plate.dm

```diff
@@ -392,7 +392,10 @@
  * Other vars such as alpha will automatically be applied with the render source
  */
 /atom/movable/screen/plane_master/proc/generate_render_relays()
-	var/relay_loc = home?.relay_loc || "CENTER"
+#if MIN_COMPILER_VERSION > 515
+	#warn Fully change default relay_loc to "1,1"
+#endif
+	var/relay_loc = home?.relay_loc || ((world.byond_version > 515) ? "1,1" : "CENTER") // As we don't have a client here, we'll use the server's version to determine a default.
```

Comment:
This is redundant code, since 516 clients will always fully regen their plane masters. We would not actually want to default based off byond version, since it could lead to unintentionally breaking for older clients. Keep the warn, remove the changes.